src/sentry/south_migrations/0377_auto__add_unique_projectteam_project.py

```diff
@@ -0,0 +1,1032 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
```

Comment:
Let's just do this as `is_dangerous = True` just to make sure since there's still a race condition. And would prefer to do it in hand since it does require a lock.